.github/workflows/ci.yml

```diff
@@ -13,7 +13,4 @@ jobs:
         run: pip install flake8
       
       - name: flake8 all repo only show erros
```

Comment:
This is the name I was referring to, it can be just left as just `flake8` (it's not true anymore after you remove the `|| true` that we only show errors, we also fail the CI if they exist).